components/automate-ui/src/app/pages/integrations/gcp-form/gcp-form.component.html

```diff
@@ -1,6 +1,7 @@
 <ng-container [formGroup]="formGroup">
 
-  <ng-container [formGroup]="formGroup.controls.credentials">
+  <!-- Unlike its siblings, gcp-form is rendered unconditionally -->
```

Comment:
@scottopherson Can you confirm whether this comment is correct? I added the comment based on the fact that the aws form and the azure form, essentially identical siblings to this, both include an `ngIf` but this one does not. So either it needs this comment, or it needs to have an `ngIf` added.

Reply:
This is correct afaik. Configuring credentials via UI is optional for all but GCP; it never needed `showInstanceCreds` method whenever that was added.